starter-project-web-react/pages/index.tsx

```diff
@@ -17,6 +18,19 @@ export async function getServerSideProps(context:any) {
         permanent: false,
       },
     }
+  }else{
+    try {
+      const res = await axios.get(
+        `${process.env.API_BASE_URL}/user/${session.id}`,
+        {headers: { "Authorization": "Bearer " + session.access_token}})
+
+      if(res.status === 200){
+        session.user = {name: res.data.fullName, email: res.data.email} 
+      }
+
+    } catch (error: any) {
+      console.log(error.message)
```

Comment:
better to not have console log in the long run. so have a plan to remove it later atleast